sdk/tables/data-tables/src/serialization.ts

```diff
@@ -28,7 +28,13 @@ type serializedType = {
 
 function serializePrimitive(value: any): serializedType {
   const serializedValue: serializedType = { value };
-  if (typeof value === "boolean" || typeof value === "string" || typeof value === "number") {
+  if (
```

Comment:
Could we write a test or two that exercises these new cases by calling `serialize` on objects with keys who's values are `null` or `undefined`?